test/functional/bundleadd/add-alias-basic.bats

```diff
@@ -0,0 +1,293 @@
+#!/usr/bin/env bats
+
+# Author: <author name>
+# Email: <author email>
+
+load "../testlib"
+
+global_setup() {
+
+	create_test_environment "$TEST_NAME"
+	create_bundle -n test-bundle1 -f /foo/test-file1 "$TEST_NAME"
+	create_bundle -n alias-bundle1 -f /foo/alias-file1 "$TEST_NAME"
+	create_bundle -n alias-bundle2 -f /foo/alias-file2 "$TEST_NAME"
+	create_bundle -n alias-bundle3 -f /foo/alias-file3 "$TEST_NAME"
+}
+
+test_setup() {
+
+        return
+
+}
+
+test_teardown() {
+
+	remove_bundle -L "$TEST_NAME"/web-dir/10/Manifest.test-bundle1
+	remove_bundle -L "$TEST_NAME"/web-dir/10/Manifest.alias-bundle1
+	remove_bundle -L "$TEST_NAME"/web-dir/10/Manifest.alias-bundle2
+	remove_bundle -L "$TEST_NAME"/web-dir/10/Manifest.alias-bundle3
+        run sudo sh -c "rm -rf "$TARGETDIR"/usr/share/defaults/swupd/alias.d/"
```

Comment:
tab and space mixing happening in test files as well.

Reply:
Yicky and should be fixed.